tests/HUBComponentImageDataBuilderTests.m

```diff
@@ -49,6 +49,7 @@ - (void)setUp
                                                                                       iconImageResolver:iconImageResolver];
     
     self.builder = [[HUBComponentImageDataBuilderImplementation alloc] initWithJSONSchema:JSONSchema iconImageResolver:iconImageResolver];
+    self.builder.bundle = [NSBundle bundleForClass:[self class]];
```

Comment:
Nit: `class` is a property on `NSObject` instances. As such dot-notation should be used according to our style guide.